consensus/hotstuff/pacemaker/pacemaker.go

```diff
@@ -203,9 +205,10 @@ func (p *ActivePaceMaker) LastViewTC() *flow.TimeoutCertificate {
 
 // Start starts the pacemaker by starting the initial timer for the current view.
 // Start should only be called once - subsequent calls are a no-op.
-func (p *ActivePaceMaker) Start() {
+func (p *ActivePaceMaker) Start(ctx context.Context) {
 	p.started.Do(func() {
```

Comment:
I feel the usage of a concurrency primitive is actually misleading here: 

Just consider the hypothetical scenario where we would call `Start` from a different goroutine than the one executing the other business-logic methods
* Then, the updates to https://github.com/onflow/flow-go/blob/207e42764f811f65861596e76e9e11ea1dcaa6af/consensus/hotstuff/pacemaker/pacemaker.go#L210 are not necessarily visible to the threads executing the business-logic methods
* Same argument for the `timeout.Controller`'s internal variables when https://github.com/onflow/flow-go/blob/207e42764f811f65861596e76e9e11ea1dcaa6af/consensus/hotstuff/pacemaker/pacemaker.go#L211

This logic only works because the event-loop [calls start](https://github.com/onflow/flow-go/blob/207e42764f811f65861596e76e9e11ea1dcaa6af/consensus/hotstuff/eventloop/event_loop.go#L91) in the routine also executing the business logic. Therefore, I would suggest to be explicit here and work with a simple boolean to not mislead anyone looking at this code into believing it might be concurrency safe.